src/lib/lumi.ts

```diff
@@ -1589,6 +1609,107 @@ export const lumiModernExtend = {
         zigbeeCommandOptions: {manufacturerCode},
         ...args,
     }),
+    lumiPreventReset: (): ModernExtend => {
+        const onEvent: OnEvent = async (type, data, device) => {
+            if (
+                // options.allow_reset ||
+                type !== 'message' ||
+                data.type !== 'attributeReport' ||
+                data.cluster !== 'genBasic' ||
+                !data.data[0xfff0] ||
+                // eg: [0xaa, 0x10, 0x05, 0x41, 0x87, 0x01, 0x01, 0x10, 0x00]
+                !data.data[0xFFF0].slice(0, 5).equals(Buffer.from([0xaa, 0x10, 0x05, 0x41, 0x87]))
+            ) {
+                return;
+            }
+            const payload = {
+                [0xfff0]: {
+                    value: [0xaa, 0x10, 0x05, 0x41, 0x47, 0x01, 0x01, 0x10, 0x01],
+                    type: 0x41,
+                },
+            };
+            await device.getEndpoint(1).write('genBasic', payload, {manufacturerCode});
+        };
+        return {onEvent, isModernExtend: true};
+    },
+    lumiClickMode: (args?: Partial<modernExtend.EnumLookupArgs>) => modernExtend.enumLookup({
+        name: 'click_mode',
+        lookup: {'fast': 1, 'multi': 2},
+        cluster: 'manuSpecificLumi',
+        attribute: {ID: 0x0125, type: 0x20},
+        description: 'Click mode for wireless button. fast: only supports single click but allows faster reponse time.' +
+            'multi: supports multiple types of clicks but is slower, because it awaits multiple clicks.',
+        entityCategory: 'config',
+        zigbeeCommandOptions: {manufacturerCode},
+        ...args,
+    }),
+    lumiSlider: (): ModernExtend => {
+        const fromZigbee: Fz.Converter[] = [{
+            cluster: 'manuSpecificLumi',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty(652)) {
+                    const actionLookup: KeyValueNumberString = {
+                        1: 'slider_single',
+                        2: 'slider_double',
+                        3: 'slider_hold',
+                        4: 'slider_up',
+                        5: 'slider_down',
+                    };
+                    return {
+                        slide_time: msg.data[561],
```

Comment:
Can we prefix these all with `action_`? Then they will be automatically cleared.

Reply:
So, that is how to do it. Got it, I'll do this now